ui/app/components/app/asset-list/asset-list.component.js

```diff
@@ -104,7 +89,6 @@ export default class AssetList extends Component {
                 name: 'Clicked Token',
               },
             })
-            selectedTokenAddress !== tokenAddress && sidebarOpen && hideSidebar()
           }}
         />
         {this.renderAddToken()}
```

Comment:
okay, these file changes just remove the triggering of sidebar hiding from the asset list component. Looks good, makes sense, and explains the below change to the container